Cargo.toml

```diff
@@ -40,20 +40,20 @@ lazy_static = "1.4.0"
 regex = "1.7.1"
 rustc_version = "0.4"
 serde = "1.0.154"
-serde_json = "1.0.94"
+serde_json = "1.0.114"
 tracing = "0.1"
-zenoh = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "main", features = [
+zenoh = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "dev/1.0.0", features = [
   "unstable", "plugins"
 ] }
-zenoh-collections = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "main" }
-zenoh-core = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "main" }
-zenoh-ext = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "main", features = [
+zenoh-collections = { version = "0.11.0-dev", git = "https://github.com/eclipse-zenoh/zenoh.git", branch = "dev/1.0.0" }
```

Comment:
In `dev/1.0.0` you should not be forced to import anything else than `zenoh` crate. Could you try to remove all crates excepts for `zenoh_backend_traits` and `zenoh-plugin-trait`?